src/components/mission.jsx

```diff
@@ -1,4 +1,37 @@
 import React from 'react';
 
-const Mission = () => <section className="mission">Mission</section>;
+const Mission = () => (
+  <section id="mission" className="mission">
+    <table className="table table-bordered table-striped">
+      <thead>
+        <tr>
+          <th scope="col">Mission</th>
+          <th scope="col">Description</th>
+          <th scope="col">Status</th>
+          <th scope="col"></th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th scope="row">Thaicom</th>
```

Comment:
- [ ]  To make the component simpler you may refactor the `<tr>` in a different component, then render it with an array  `map` method. This will make this component shorter and maintainable